src/libradosstriper/RadosStriperImpl.cc

```diff
@@ -12,6 +12,8 @@
  *
  */
 
+#include <boost/algorithm/string/replace.hpp>
```

Comment:
the commit message should look like:
```
Fixes: http://tracker.ceph.com/issues/20240
SIgned-off-by: Stan K <stan's email>
```
and we need to have Stan's email, assuming he agrees to contribute his patch to Ceph.